point_viewer_proto_rust/src/proto.proto

```diff
@@ -75,23 +75,51 @@ message NodeId {
   int64 deprecated_index = 2;
 }
 
-message Node {
-  enum PositionEncoding {
+enum PositionEncoding {
     INVALID = 0;
     Uint8 = 1;
     Uint16 = 2;
     Float32 = 3;
     Float64 = 4;
-  }
+}
 
+
+message Node {
   PositionEncoding position_encoding = 2;
   int64 num_points = 3;
   NodeId id = 4;
 }
 
+enum AttributeType{
+    I64 = 0;
```

Comment:
0 is reserved for unset (https://github.com/lyft/avsoftware/blob/master/STYLE.md#protobuf-coding-style) or unspecified (https://developers.google.com/protocol-buffers/docs/style#enums), so we should add it here...